.github/workflows/backport.yml

```
@@ -28,16 +33,30 @@ jobs:
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      ## GH token/ app token/PAT
+      # gh api or gh cli
       - name: Cherry-pick to backport branch
         run: |
           git config user.name "adela-domokosova"
           git config user.email "adela.domokosova@gmail.com"
+               
+          git checkou -B backport/pr-${{ env.PR_NUMBER }}
+          git clone 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)